fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/app/Main.java

```diff
@@ -901,7 +902,9 @@ protected void listTenants() {
                 List<TenantInfo> tenants = adapter.runStatement(rtListGetter);
 
                 System.out.println(TenantInfo.getHeader());
-                tenants.forEach(t -> System.out.println(t.toString()));
+                tenants.forEach(System.out::println);
+            } catch(UndefinedNameException x) {
+                System.out.println("The FHIR_ADMIN schema appears not to be deployed with the TENANTS table");
```

Comment:
@prb112 
late to the party on this one, but shouldn't this use `schema.getAdminSchemaName()` instead of FHIR_ADMIN here?  I wouldn't advise folks to change the admin schema name but I think you mentioned that we do support it?